src/client/WOLF.js

```diff
@@ -200,16 +201,28 @@ class WOLF extends EventEmitter {
       }
     }
 
+    if (categoryIds) {
+      categoryIds = Array.isArray(categoryIds) ? categoryIds : [categoryIds];
+
+      categoryIds.forEach((categoryId) => {
+        if (!validator.isValidNumber(categoryId)) {
+          throw new WOLFAPIError('categoryId must be a valid number', { categoryId });
+        } else if (validator.isLessThanOrEqualZero(categoryId)) {
+          throw new WOLFAPIError('categoryId cannot be less than or equal to 0', { categoryId });
+        }
```

Comment:
Should validate that IDs exist in client.channel.category.list(language??client.config.get('framework.language))